L1Trigger/L1THGCal/interface/backend_emulator/BatcherSorter.h

```diff
@@ -21,11 +21,11 @@ typedef std::vector<unsigned> adress_t;         // Address Pointer for sorter &
 
 namespace l1thgcfirmware {
 
-  class HGCalStage1SortingAlg_SA {
+  class BatcherSorter {
```

Comment:
I don't have a strong opinion on that. I would say that the `HGCalLayer/Stage` prefix should be fixed for plugins (i.e. Producers, Processors, Wrappers), but for implementations it would be more lightweight to remove the prefix. And we could eventually differentiate Stage 1 and Stage 2 implementations in two directories, e.g. `backend_stage1` and `backend_stage2` to make things better organized